modules/@ergonode/category-trees/src/locales/pl_PL.json

```diff
@@ -0,0 +1,30 @@
+{
+  "categoryTree": {
+    "page": {
+      "title": "",
+      "head": "",
+      "addButton": ""
+    },
+    "form": {
+      "noCategoryTreeButton": "IDŹ DO DRZEWA KATEGORII",
+      "noVisibleCategoriesInTreePlaceholderTitle": "Brak kategorii",
+      "noVisibleCategoriesInTreePlaceholderSubtitle": "Nie ma zaznaczonych kategorii dla wybranych kryteriów."
+    },
+    "advancedFilter": {
+      "label": "Drzewo kategorii"
+    },
+    "grid": {
+      "placeholderTitle": "Brak drzew kategorii",
+      "placeholderSubtitle": "W systemie nie ma drzew kategorii, stwórz pierwsze."
+    },
+    "messages": {
+        "deleteTitle": "",
```

Comment:
it should be an empty string?

Reply:
I was copying it, sorry for missing. I've added missing translations.